test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -387,6 +387,51 @@ public void CanReadAndWriteExcel()
 
         }
 
+        [Test]
+        public void CanReadExcelAsStrings()
+        {
+
+            string openPath = Path.Combine(TestDirectory, @"core\excel\ReadExcelAsStrings.dyn");
+            ViewModel.OpenCommand.Execute(openPath);
+
+            var filename = ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<DSCore.File.Filename>();
+
+            // remap the filename as Excel requires an absolute path
+            filename.Value = filename.Value.Replace(@"..\..\..\test", TestDirectory);
+
+            // watch displays the data from the Read node
+            var watch = ViewModel.Model.CurrentWorkspace.GetDSFunctionNodeFromWorkspace("Excel.ReadFromFile");
+
+           ViewModel.HomeSpace.Run();
+
+            Assert.IsTrue(watch.CachedValue.IsCollection);
+            var list = watch.CachedValue.GetElements();
+
+            Assert.AreEqual(5, list.Count());
+
+            // single column - 1, "word", 2, 3, "palabra"
+            Assert.IsTrue(list[0].IsCollection);
+            var rowList = list[0].GetElements();
+            Assert.AreEqual("1", rowList[0].Data);
```

Comment:
We have added a new default argument to the `Excel.ReadFromFile` method called `readAsStrings`. If this argument is `true`, the data read from Excel would be in the form of strings no matter if they are strings or numbers.